assets/prometheus-k8s/rules.yaml

```diff
@@ -345,6 +345,14 @@ spec:
       record: cluster:node_instance_type_count:sum
     - expr: sum(etcd_object_counts) BY (instance)
       record: instance:etcd_object_counts:sum
+    - expr: max(etcd_object_counts{resource=~"builds\.build\.openshift\.io|deploymentconfigs\.apps\.openshift\.io|images\.image\.openshift\.io|statefulsets\.apps|deployments\.apps|cronjobs\.batch|jobs\.batch|pods|persistentvolumeclaims|persistentvolumes|services|namespaces|routes\.route\.openshift\.io|ingresses\.networking\.k8s\.io|horizontalpodautoscalers\.autoscaling|users\.user\.openshift\.io|machinesets\.machine\.openshift\.io|resourcequotas"})
```

Comment:
I get the following error when trying to perform this query:

> Error executing query: parse error at line 1, char 34: unknown escape sequence U+002E '.'

So this might be the reason the tests are failing consistently.